indico/modules/events/templates/display/conference.html

```diff
@@ -49,6 +45,10 @@
                             {% if event.address %}
                                 <div class="address nohtml">{{ event.address }}</div>
                             {% endif %}
+
+                            {% if event.map_url or (event.room and event.room.map_url) %}
```

Comment:
What about adding a property to `Event` that returns either the custom map url or the room's map url?